openmc/statepoint.py

```diff
@@ -92,13 +91,8 @@ class StatePoint(object):
     """
 
     def __init__(self, filename):
-        if filename.endswith('.h5'):
-            import h5py
-            self._f = h5py.File(filename, 'r')
-            self._hdf5 = True
-        else:
-            self._f = open(filename, 'rb')
-            self._hdf5 = False
+        import h5py
```

Comment:
Perhaps ``import h5py`` should be declared at the top of this file since HDF5 and ``h5py`` are no longer optional dependencies?